apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -140,11 +168,12 @@ defmodule AndiWeb.EditLiveView do
 
     if changeset.valid? do
       changes = Ecto.Changeset.apply_changes(changeset)
+      dataset = InputConverter.restruct(changes, original_dataset)
```

Comment:
Was this just a boy scout change?

Reply:
Yeah, Credo was upset about it so I went ahead and changed it.